src/app/store/task/task.reducer.ts

```diff
@@ -0,0 +1,36 @@
+import { createReducer, on, Action } from "@ngrx/store"
+import AppState from "../app.state"
+import * as taskActions from "./task.actions"
+import { Task } from "../../models/task.model"
+import { initialState } from "../app.state"
+
+
+
+
+const _taskReducer = createReducer(
```

Comment:
would be good if you had initalState declared for each reducer, so initialState for tasks and users